test/learn.jl

```diff
@@ -142,7 +142,7 @@ end
         @testplot confusion_row
 
         confusion_col = plot_confusion_matrix(plot_data["confusion_matrix"],
-                                              plot_data["class_labels"], :Row)
+                                              plot_data["class_labels"], :Column)
```

Comment:
...I'm kind of surprised that this didn't break the percy test?

Reply:
Yeah, that was an embarrassing oversight :D I approved the percy build. so now it uses this as the baseline